### PR TITLE
Task 1

### PR DESCRIPTION
**Pull request by kejno**
[TASK 1](https://epa.ms/nodejs19-hw1)

### CODE REVIEW

File: task1.2.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import fs from 'fs'
+import { csv } from 'csvtojson'
+
+const csvFilePath = path.join(__dirname, 'csv', 'input.csv')
+const outputFilePath = path.join(__dirname, 'output', 'output.txt')
+
+let readableStream = fs.createReadStream(csvFilePath, 'utf8');
+let writeableStream = fs.createWriteStream(outputFilePath);
+
+readableStream.on('data', async (chunk) => {
```
Comment (andriikukushkin) [inline task1.2.js]: Using pipeline you also can filter a data.

File: package.json [hunk 2]
```diff
@@ -4,7 +4,9 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "task1": "nodemon task1.1.js",
```
Comment (andriikukushkin) [inline package.json:7]: I can't execute `npm run task1` and `npm run task2` commands. You missed something to add.

File: task1.1.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
+function reverseString(data){ 
+    const reverseStr= data.toString().split('').reverse().join('');
+    process.stdout.write(reverseStr.trim() + '\n\n');
+}
+
+process.stdin.on('data', reverseString);
+
+
+
+// const readline = require('readline')
```
Comment (andriikukushkin) [inline task1.1.js]: Why you commented this?

**Review by andriikukushkin — CHANGES_REQUESTED** [review]
Try to filter data using pipeline. Also please exit from process when file processing is done (like once output file is ready program should be closed).
Please check other comments too.

File: package.json (hunk 2 above)
Comment (kejno) [reply package.json:7]: Fixed. I need install nodemon in dev dep.

File: task1.1.js (hunk 3 above)
Comment (kejno) [reply task1.1.js]: It was second decision. I deleted it

File: task1.2.js (hunk 1 above)
Comment (kejno) [reply task1.2.js]: I created pipeline.js where I implemented stream though pipeline

**Review by andriikukushkin — APPROVED** [review]

**Pull request closed without merging**